themes/wdfn_theme/src/styles/main.scss

```diff
@@ -1,4 +1,30 @@
-@import '../../node_modules/wdfn-viz/dist/sass/wdfnviz';
+@import '../../node_modules/wdfn-viz/dist/sass/theme';
+@import '../../node_modules/wdfn-viz/dist/sass/uswds';
+
+$usgs-blue: #00264c;
+
+@import '../../node_modules/wdfn-viz/dist/sass/partials/header';
+@import '../../node_modules/wdfn-viz/dist/sass/partials/footer';
```

Comment:
Why was this necessary?  I pulled the code down and restored the original single import and I didn't see any differences on the roadmap page, so I would like the original single import restored.

Reply:
the roadmap page isn't the problem.  Try one of the rapidocs pages off of api-docs